pymhf/gui/gui.py

```diff
@@ -260,7 +260,8 @@ def on_update(_, app_data, user_data):
                         source=tag,
                         callback=on_update,
                         user_data=(cls, variable),
-                        **extra_args,
```

Comment:
these changes are not required. If you want to have `on_enter=True`, then specify it in the `STRING` variable decorator as an extra arg and it will come through to here (cf. lines 257 & 258)